user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Configuring_external_authentication_via_an_identity_provider_using_SAML.md

```diff
@@ -269,6 +271,26 @@ Once you have established a trust relationship between DataMiner (i.e. the servi
 
    It is now possible to log in to DataMiner with any of the Azure AD user accounts you have added, using either the domain and username (DOMAIN\\user) or the email address.
 
+#### Configuring DataMiner to check if the client secret is about to expire
+
+From DataMiner 10.3/10.2.9 onwards, it is possible to configure DataMiner to check Azure Active Directory for the expiration of the client secret when authenticating (only when application querying is used). When the client secret will expire in less than a week, a warning will be shown in the alarm console.
+
+For this 2 extra configurations are required:
+
+1. On Azure's side an extra permission: Microsoft Graph \> Application.Read.All – Application – Read applications
+1. On DataMiner's side an extra guid in DataMiner.xml: the **Object ID**. This GUID can be found on the same page where the **Client ID** and **Tenant ID** are found. You can find these fields on the root page of the application: *Azure Active Directory > App registrations > [your application name]*.
```

Comment:
```suggestion
1. In DataMiner.xml, add the **Object ID**. Like the **Client ID** and **Tenant ID**, you can find this GUID on the root page of the application: *Azure Active Directory > App registrations > [your application name]*.
```
Note that this step may need to be expanded a bit, as to my knowledge this will also require a DataMiner restart.